server/profile/profile.go

```diff
@@ -22,6 +24,22 @@ type Profile struct {
 	DSN string `json:"-"`
 	// Version is the current version of server
 	Version string `json:"version"`
+	// Feat is the feature set of server, split by comma
+	Feat string `json:"feature"`
+}
+
+const (
+	FeatSSO       Feature = "SSO"
```

Comment:
* Using full name will be more easy to understand;
* Using `FeatureType` instead of `Feature`

And let's move them into a new file `server/profile/feature.go`

```suggestion
	FeatureSSO       FeatureType = "SSO"
```

Reply:
![image](https://user-images.githubusercontent.com/31177490/220270263-23830782-3c58-4c80-9115-4773ffa28507.png)

Reply:
i'd prefer `server/profile/feature.go` rather than `server/feature.go`